tasks/main.yml

```diff
@@ -19,3 +19,9 @@
 
 - include_tasks: agent6-win.yml
   when: not datadog_agent5 and ansible_os_family == "Windows"
+
+- name: add "{{ datadog_user }}" user to additional groups - Linux
```

Comment:
why not add this to agent6-linux.yaml instead ?

Reply:
Yes, if we want to provide this for agent 6 only, works for me.